code/drasil-example/Drasil/GamePhysics/DataDefs.hs

```diff
@@ -245,6 +246,56 @@ chaslesThmDesc = foldlSent [S "The linear", (phrase QP.velocity),
   (phrase rOB) `sC` (ch rOB), 
   (sParen $ Sy $ unit_symb rOB)]
 
+---------------DD10 Impulse(Vector)-----------------------------------------------------------------------
+impulseVDD :: DataDefinition
+impulseVDD = ddNoRefs impulseV impulseVDeriv "impulseV"
+ [impulseVDesc, makeRef2S assumpOT]
+
+impulseV :: QDefinition
+impulseV = mkQuantDef QP.impulseV impulseVEqn
+
+impulseVEqn :: Expr
+impulseVEqn =   (sy QPP.mass) * (sy QP.chgInVelocity)
+
+impulseVDesc :: Sentence
+impulseVDesc = foldlSent [S "An", (phrase QP.impulseV), (ch impulseV), S "occurs when a", 
+  (phrase QP.force), (ch QP.force), S "acts over a body over an interval of",(phrase QP.time),
+  S "Derivation of", (phrase QP.impulseV)]
+
+impulseVDeriv :: Derivation
+impulseVDeriv = (weave [impulseVDerivSentences, map E impulseVDerivEqns]) 
+
+impulseVDerivSentences :: [Sentence]
+impulseVDerivSentences = map foldlSentCol [impulseVDerivSentence1, 
+ impulseVDerivSentence2, impulseVDerivSentence3]  
+
+impulseVDerivSentence1 :: [Sentence]
+impulseVDerivSentence1 = [S " Derivation of", (phrase QP.impulseV), S "-", 
+  S "Newton's second law of motion states"]
+
+impulseVDerivSentence2 :: [Sentence]
+impulseVDerivSentence2 = [S "Rearranging "] 
+
+impulseVDerivSentence3 :: [Sentence]
+impulseVDerivSentence3 = [S "Integrating the right hand side "] 
+
+impulseVDerivEqn1 :: Expr
+impulseVDerivEqn1 = (sy QP.force) $= (sy QPP.mass) * (sy QP.acceleration) 
+                    $= (sy QPP.mass) * (deriv(sy QP.velocity) QP.time) 
+
+impulseVDerivEqn2 :: Expr
+impulseVDerivEqn2 = (defint (eqSymb timeT)(sy time_1)(sy time_2)(sy QP.force))
+                     $= sy QPP.mass * defint (eqSymb QP.velocity)(sy velo_1)(sy velo_2)1
```

Comment:
Do you really want to integrate `1` here?

Reply:
No, I dont want to integrate but I had to put '1'  in there in order to provide 4 arguements required by the defint function. Is there another way to do this? like some 'empty' expression?

Reply:
What are you actually trying to write down? (Give me the 'math').

In a way, integrating the constant 1 is sometimes actually the right thing - it's just that we tend to abuse notation and elide it. But I'm not sure if this is such a case.

Reply:
Here is what the actual formula looks like, everything without the '1'
<img width="224" alt="image" src="https://user-images.githubusercontent.com/43192745/58970783-18a0bc80-8788-11e9-8a1b-b1efcc55f9df.png">

But I had to put '1' there as a placeholder in order to be able to use 'defint' function. I don't think it makes any difference in the calculation though.

Reply:
It does not, and is the correct change to make. Please file an issue to record that we might want to pretty-fy the output of integrating over `1`.

Reply:
> It does not, and is the correct change to make. Please file an issue to record that we might want to pretty-fy the output of integrating over 1.

#1539